src/components/Technologies/Technologies.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { TechnologiesSection, TechnologiesImage, TechnologiesTitle, TechnologiesSubtitle, TechnologiesTextsContainer, TechnologiesImagesContainer, TechnologiesLink } from './Technologies.styles'
+import { technologiesImages } from '@/constants/technologies-images'
+
+export const Technologies: React.FC = () => {
+  const { t } = useTranslation()
+
+  return (
```

Comment:
I'm starting to be totally in love 💘 
The amazing thing is all components looks the same, that's just what we want!! you're doing a super work!!